src/gui/transferlistfilterswidget.cpp

```diff
@@ -488,7 +488,8 @@ void TrackerFiltersList::addItem(const QString &tracker, const QString &hash)
         trackerItem = new QListWidgetItem();
         trackerItem->setData(Qt::DecorationRole, GuiIconProvider::instance()->getIcon("network-server"));
 
-        downloadFavicon(QString("http://%1/favicon.ico").arg(host));
+        if(Preferences::instance()->isLoadFaviconsEnabled())
```

Comment:
Good point. But this will be a "tough" one to implement cleanly. All the classes in that file are simple enough.
@slin31 here is what you have to look for Preferences class has a `changed()` signal. You need to connect to that and chance your local value(which is created/cached in the constructor of TrackerFiltersList probably).